src/signals/incident-management/containers/Dashboard/components/AreaChart.tsx

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2023 Gemeente Amsterdam
+import vegaEmbed from 'vega-embed'
+
+import { getAreaChart } from '../charts'
+import { ModuleTitle } from '../components/ModuleTitle'
+
+/** Mock data */
+const mockValues = [
+  { a: '01 Jan 2012 23:00:00', b: 48 },
+  { a: '02 Jan 2012 23:00:00', b: 34 },
+  { a: '03 Jan 2012 23:00:00', b: 44 },
+  { a: '04 Jan 2012 23:00:00', b: 40 },
+  { a: '05 Jan 2012 23:00:00', b: 48 },
+  { a: '06 Jan 2012 23:00:00', b: 45 },
+  { a: '07 Jan 2012 23:00:00', b: 47 },
+]
+
+const getMaxDomain = (values: typeof mockValues) => {
+  let max = 0
```

Comment:
Would consider a reduce for this one, typical use case.

Reply:
outdated